forem/forem
# Reset password design and Signup page update for invite-only forems.

## What type of PR is this? (check all applicable)


## Description

- [x] Implement password reset page designs for desktop and mobile. [Figma](https://www.figma.com/file/u7VcNG4pjbn7K01V8JrFWo/Member-Onboarding-Process?type=design&node-id=3182-256&mode=design&t=U3rkSHkbhtcf0YIK-0)
- [x]  Update signup page for invite only forem [Figma](https://www.figma.com/file/u7VcNG4pjbn7K01V8JrFWo/Member-Onboarding-Process?node-id=2733%3A20022&mode=dev)
- [x]  The logo at the top of signup/login pages should be of 48px height and width should be auto.
- [x]  Click on the logo on these pages should redirect to home page.
- [x] Minor fix of design if there is only email based login. (Check Before/After table images below)

## Related Tickets & Documents



- Closes #20142

## QA Instructions, Screenshots, Recordings

**How to test**?
1. Sign in as admin
2. Visit `/admin/customization/config` -> Authentication and select `Invite only` check for testing
3. In incognito mode visit following pages: `/users/password/new`, `/enter`

### Output
<img width="1512" alt="Screenshot 2023-09-20 at 6 19 46 PM" src="https://github.com/forem/forem/assets/9396084/14899686-39f0-487f-a6d1-6d8cd5769c48">

<img width="1512" alt="Screenshot 2023-09-20 at 11 19 38 PM" src="https://github.com/forem/forem/assets/9396084/fa1b47b6-f1a0-4ba7-a5e4-28fac07d7235">
<img width="380" alt="Screenshot 2023-09-20 at 11 19 55 PM" src="https://github.com/forem/forem/assets/9396084/2a0eaa66-f8b8-4e54-8ed0-422e0f1ada3d">


| Before  | After |
| ------------- | ------------- |
| <img width="764" alt="Screenshot 2023-09-20 at 6 21 25 PM" src="https://github.com/forem/forem/assets/9396084/a9b68dc4-1926-4e5a-9ecb-ff2efbd94db3"> | <img width="703" alt="Screenshot 2023-09-20 at 6 23 44 PM" src="https://github.com/forem/forem/assets/9396084/86f9922d-427e-4f5e-a166-42bd7dee4a3d"> |



### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._
